Polyfills/AbortController/Source/AbortController.h

```diff
@@ -10,6 +10,7 @@ namespace Babylon::Polyfills::Internal
     class AbortController final : public Napi::ObjectWrap<AbortController>
     {
     public:
+        static constexpr auto JS_ABORT_CONTROLLER_CONSTRUCTOR_NAME = "AbortController";
```

Comment:
Why does this need to be public?

Reply:
It doesn't I can make it private or move inside the cpp file.

Reply:
I would recommend inside the cpp file.

Reply:
I moved some of the other definitions into their respective cpp file as well, but AbortSignal and URLSearchParam are used outside so they needed to remain public.